sysbuild/CMakeLists.txt

```diff
@@ -760,6 +767,10 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_post_cmake)
     include(${ZEPHYR_NRF_MODULE_DIR}/cmake/sysbuild/bootconf.cmake)
   endif()
 
+  if(SB_CONFIG_BOOTLOADER_MCUBOOT)
```

Comment:
properly guard here for the device and Kconfig from the other file so it's only included if used